traveler_core_kit/src/main/java/com/guestlogix/travelercorekit/utilities/JsonObjectMapper.java

```diff
@@ -25,4 +36,45 @@ public void onHandleResponse(InputStream stream) {
             callback.onError(new ObjectMappingError(objectMappingFactory, e));
         }
     }
+
+    private InputStream logNetworkResponse(InputStream inputStream) {
```

Comment:
Rename this method into something more related to the class that has it. Meaning, your class JsonMapper is in charge of mapping data to models, as such its log method should be named accordingly.

Reply:
renemaed it to logRawJson ;)